profiling/src/profiling/mod.rs

```diff
@@ -750,6 +751,12 @@ impl Profiler {
                     key: "exception type",
                     value: LabelValue::Str(exception.clone().into()),
                 });
+
+                labels.push(Label {
+                    key: "exception message",
+                    value: LabelValue::Str(message.clone().into()),
```

Comment:
Is the clone necessary? `message` was added by value to the function signature, so it's owned already and I don't see any other uses of it in the function.

Reply:
You see it right, the `clone()` is not needed, I removed it. Thanks for looking!